falcon/util/uri.py

```diff
@@ -246,7 +246,8 @@ def decode(encoded_uri):
         return decoded_uri.decode('utf-8', 'replace')
 
 
-def parse_query_string(query_string, keep_blank_qs_values=False):
+def parse_query_string(query_string, keep_blank_qs_values=False,
+                       auto_parse_query_string_lists=True):
```

Comment:
This param name could be shortened a little bit by omitting "query_string", since that is already implied by the function name.